src/main/resources/interface-specification.yml

```diff
@@ -3022,6 +3022,7 @@ paths:
             application/problem+json:
               schema:
                 $ref: '#/components/schemas/Problem'
+
```

Comment:
please do not add these new lines, unless we all agree to use this new formatting style

Reply:
Sorry, probably it was a new line I forgot to delete, I'll push a quick commit in order to fix it!